.github/workflows/ci.yml

```diff
@@ -44,14 +45,10 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
-      - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af
+      - name: Build
+        uses: ./.github/actions/build
```

Comment:
I copied the way `ci` → `types` works but let me know if I did this right.

Reply:
Unfortunately, this is is not working as you’d think. With and without your changes the build will run first and pass no matter if there’s formatting issues.

But the CI workflow will fail in both cases in the same way.

We can run prettier *before* the build and require prettier to pass before we do a build, but that would extend CI times by 40s or so.

We can chat to find a better solution! ✌️

Reply:
I'm not sure I follow but sure we can chat about it